packages/react-components/src/components/VideoGallery/LocalScreenShare.tsx

```diff
@@ -44,13 +56,38 @@ export const LocalScreenShare = React.memo(
     if (!localParticipant || !localParticipant.isScreenSharingOn) {
       return null;
     }
+    const localScreenSharingNotification = (
```

Comment:
We use notification to means that ufd alert UI, is the same UI or the old UI were the tile shows placeholder text and icon?
If its the same we should be using the notification component, if its the old UI change the naming here

Reply:
Your choice, but notification is now a different component we ship so we should avoid that, one option would be `localScreenSharingPlaceholder`